.travis.yml

```diff
@@ -6,11 +6,7 @@ go:
   - 1.6
 
 before_install:
-  - go get github.com/golang/mock/gomock
-  - go get github.com/golang/mock/mockgen
-  - go get github.com/onsi/gomega
-  - go get github.com/onsi/ginkgo
-  - go get golang.org/x/tools/cmd/cover
+  - make dependencies
```

Comment:
ARG, ok, I hit this too.

I think it's that the test command in Makefile should include a go install github.com/golang/mock/mockgen

I haven't been able to figure out how it currently works.